x-pack/legacy/plugins/maps/public/angular/services/gis_map_saved_object_loader.js

```diff
@@ -7,11 +7,12 @@
 import './saved_gis_map';
 import { uiModules } from 'ui/modules';
 import { SavedObjectLoader, SavedObjectsClientProvider } from 'ui/saved_objects';
+import { npStart } from '../../../../../../../src/legacy/ui/public/new_platform';
 
 const module = uiModules.get('app/maps');
 
 // This is the only thing that gets injected into controllers
-module.service('gisMapSavedObjectLoader', function (Private, SavedGisMap, kbnUrl, chrome) {
+module.service('gisMapSavedObjectLoader', function (Private, SavedGisMap) {
   const savedObjectClient = Private(SavedObjectsClientProvider);
-  return new SavedObjectLoader(SavedGisMap, kbnUrl, chrome, savedObjectClient);
+  return new SavedObjectLoader(SavedGisMap, savedObjectClient, npStart.core.chrome);
```

Comment:
could we use the saved object client from `npStart` instead? One piece of angular less.

Reply:
of course! done!